File: netflix2/java/src/main/java/repository/UserDao.java
package repository;

import source.DB;

import java.sql.ResultSet;
import java.sql.SQLException;

public class UserDao {

    /**
     * Register a user to the system.
     * The user identifier is unique.
     *
     * @param userId identifier of a user
     * @param password password
     * @return true if the user has been successfully registered, false otherwise
     */
    public static void register(String userId, String password) {
        DB.insert(String.format("INSERT INTO USERS(ID, PASSWORD) values('%s', '%s')", userId, password));
    }

    /**
     * Check if a user
     * @param userId identifier of a user
     * @param password password
     * @return true if the user exists, false otherwise
     */
    public static boolean exist(String userId, String password) {
        ResultSet rs = DB.execute(String.format("SELECT count(1) as total FROM USERS where id='%s' AND password='%s'", userId, password));
        try {
            rs.next();
            return rs.getInt("TOTAL") > 0;
        } catch (SQLException e) {
            e.printStackTrace();
            return true;
        }
    }

}

File: netflipster/DotNetCore/Netflipster/Repository/UserDao.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Netflipster.Repository
{
    public class UserDao
    {
        /// <summary>Register a user to the system.</summary>
        /// <param name="name">identifier of a user</param>
        /// <param name="password">password</param>
        /// <returns>true if the user has been successfully registered, false otherwise</returns>
        /// <remarks>The user identifier is unique.</remarks>
        public static bool Register(string name, string password)
        {
            return DB.Insert($"INSERT INTO USERS(NAME, PASSWORD) values('{name}', '{password}')");
        }

        /// <summary>Check if a user exists and get her identifier</summary>
        /// <param name="name">identifier of a user</param>
        /// <param name="password">password</param>
        /// <returns>the user identifier if the user exists; empty string otherwise.</returns>
        public static string FindId(string name, string password)
        {
            using (var cnx = DB.GetConnection())
            {
                using (var rs =
                    DB.ExecuteQuery($"SELECT ID FROM USERS where NAME='{name}' AND password='{password}'", cnx))
                {
                    if (rs.Read())
                    {
                        return rs.GetInt64(0).ToString();
                    }

                    return string.Empty;
                }
            }
        }
    }
}
